### DISCUSSION TITLE
a breaking change in 3.4.0

### DISCUSSION THREAD

**Discussion by cuyl** [kamilkisiela/graphql-config#713] [General]
I found a breaking change in [3.4.0](https://www.npmjs.com/package/graphql-config/v/3.4.0), it changes supported node version from`^10.17.0 || ^12.0.0 || >= 13.7.0` to `^12.20 || >= 14.13` caused by dependency `graphql-config -> @graphql-tools/url-loader -> extract-files`.

**Comment by ardatan** [comment]
I think you can ignore it with `--ignore-engines` flag. Sometimes it is hard to deal with inner dependencies because if we go back to the old version of that dependency, `ws` vulnerability thingy will pop up. 
Since Node 10 is no more LTS, maybe it is a good time for you to upgrade to Node 12

**Reply by cuyl** [reply]
I've fixed the version to 3.3.0, maybe you can bump a major version as well.
